rust-peer/fly.toml

```diff
@@ -0,0 +1,28 @@
+# fly.toml app configuration file generated for universal-connectivity-rust-peer on 2023-09-24T16:20:25+10:00
+#
+# See https://fly.io/docs/reference/configuration/ for information about how to use this file.
+#
+
+app = "universal-connectivity-rust-peer"
+primary_region = "syd"
+
+[experimental]
+# Resolve the special `fly-global-services` address before starting the container, see https://fly.io/docs/app-guides/udp-and-tcp/#the-fly-global-services-address.
+cmd = ["/bin/sh", "-c", "set -ex; LISTEN_ADDRESS=$(getent hosts fly-global-services | cut -d' ' -f1) && rust-libp2p-webrtc-peer"]
+
+[env]
+EXTERNAL_ADDRESS="149.248.215.31" # TODO: Remove this in favor of an AutoNAT setup.
```

Comment:
Would AutoNAT even work when deployed to Fly.io? In my experience running Kubo on fly.io, their overlay network messes with AutoNAT and results in the wrong IP, but it's been a while since I actually tried it and I didn't dive too deep.